dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaHostRamChartTask.java

```diff
@@ -0,0 +1,105 @@
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ClouderaHostRamChartTask extends AbstractClouderaManagerTask {
+  private static final Logger LOG = LoggerFactory.getLogger(ClouderaCMFHostsTask.class);
+  private final ObjectMapper objectMapper = new ObjectMapper();
+
+  private final ClouderaTimeSeriesQueryBuilder tsQueryBuilder;
+
+  private static final DateTimeFormatter isoDateTimeFormatter =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+
+  private static final String TS_CPU_QUERY_TEMPLATE =
+      "select swap_used, physical_memory_used, physical_memory_total, physical_memory_cached, physical_memory_buffers where entityName = \"%s\"";
+
+  public ClouderaHostRamChartTask(ClouderaTimeSeriesQueryBuilder tsQueryBuilder) {
+    super(buildOutputFileName(tsQueryBuilder.getIncludedLastDays()));
+    this.tsQueryBuilder = tsQueryBuilder;
+  }
+
+  @Override
+  protected Void doRun(
+      TaskRunContext context, @Nonnull ByteSink sink, @Nonnull ClouderaManagerHandle handle)
+      throws Exception {
+    CloseableHttpClient httpClient = handle.getHttpClient();
+
+    List<ClouderaHostDTO> hosts = handle.getHosts();
+    if (hosts == null) {
+      throw new MetadataDumperUsageException(
+          "Cloudera hosts must be initialized before RAM charts dumping.");
+    }
+
+    final String timeSeriesAPIUrl = buildTimeSeriesUrl(handle.getApiURI().toString());
+    try (Writer writer = sink.asCharSink(StandardCharsets.UTF_8).openBufferedStream()) {
+      for (ClouderaHostDTO host : handle.getHosts()) {
+        String ramPerHostQuery = tsQueryBuilder.getQuery(host.getId());
+        LOG.debug(
+            "Execute RAM charts query: [{}] for the host: [{}].", ramPerHostQuery, host.getName());
+
+        URIBuilder uriBuilder = new URIBuilder(timeSeriesAPIUrl);
+        uriBuilder.addParameter("query", ramPerHostQuery);
+        uriBuilder.addParameter("desiredRollup", tsQueryBuilder.getTsAggregation().toString());
+        uriBuilder.addParameter("mustUseDesiredRollup", "true");
+        uriBuilder.addParameter("from", buildISODateTime(tsQueryBuilder.getIncludedLastDays()));
+
+        try (CloseableHttpResponse chart = httpClient.execute(new HttpGet(uriBuilder.build()))) {
+          JsonNode jsonNode = objectMapper.readTree(chart.getEntity().getContent());
+          writer.write(jsonNode.toString());
+          writer.write('\n');
+        }
+      }
+    }
+    return null;
+  }
+
+  private String buildISODateTime(int deltaInDays) {
+    ZonedDateTime dt =
+        ZonedDateTime.of(LocalDateTime.now().minusDays(deltaInDays), ZoneId.of("UTC"));
+    return dt.format(isoDateTimeFormatter);
+  }
+
+  private String buildTimeSeriesUrl(String apiUri) {
+    return apiUri + "/timeseries";
+  }
+
+  private static String buildOutputFileName(int includedLastDays) {
+    return String.format("cmf-host-ram-%sd.jsonl", includedLastDays);
```

Comment:
I missed it initial, but we do not use `cmf` api here and in `ClouderaClusterCPUChartTask` as well. So, we do not need the prefix in file name.
The idea behind `cmf` prefix is to mark files which we fill with data collected from `cmf` endpoint. 
For charts we use documented (except exact query) API, so let's remove the prefix.